docs/KuberntesTargetFilter.md

```diff
@@ -0,0 +1,24 @@
+# Kubernetes target filtering
+
+The config parameter `target_discovery.filter` should be used to filter in the targets that Prometheus will scrape. Current conditions are filtering by `label` and `annotation`. All conditions are applied using an `AND` operation.
```

Comment:
would make sense to move as well some of the info to the chart Readme? Or at least add some links pointing here?